src/object/object_primitive.c

```diff
@@ -9541,7 +9541,7 @@ pr_midxkey_add_elements (DB_VALUE * keyval, DB_VALUE * dbvals, int num_dbvals, s
   else
     {
       /* bound bits */
-      (void) pr_midxkey_init_boundbits (bound_bits, bitmap_size);
+      (void) pr_midxkey_init_boundbits (bound_bits, num_dbvals);
```

Comment:
Since we have already calculated bitmap_size,
calculating it once more in pr_midxkey_init_boundbits is redundant.
```suggestion
      MIDXKEY_BOUNDBITS_INIT (bound_bits, bitmap_size);
```

Reply:
Good point!
I will fix it. thanks.